osu.Game/Overlays/SkinEditor/SkinEditorOverlay.cs

```diff
@@ -316,7 +316,7 @@ protected override void LoadComplete()
                 base.LoadComplete();
 
                 if (!LoadedBeatmapSuccessfully)
-                    Scheduler.AddDelayed(this.Exit, 3000);
+                    Scheduler.AddDelayed(this.Exit, RESULTS_DISPLAY_DELAY);
```

Comment:
Why reuse a results const? What does one have to do with the other?

Reply:
Also why was this shortened to begin with?

Reply:
The delay was way too awkwardly long from a ux perspective, given it's a completely blank screen with only the background. For comparison, when trying to actually play a map with 0 objects, the delay before returning to song select isn't 3s like here either.